solarforecastarbiter/reports/templates/template.md

```diff
@@ -24,10 +24,27 @@ Contents:
   * [Observations and forecasts](#observations-and-forecasts)
   * [Data validation](#data-validation)
 * [Metrics](#metrics)
-  * [Total analysis period](#total-analysis-period)
-  * [Monthly](#monthly)
-  * [Daily](#daily)
-  * [Hourly](#hourly)
+  {% if 'total' in figures.keys() %}
+  * [Total period analysis](#total-period-analysis)
+  {% endif %}
+  {% if 'month' in figures.keys() %}
+  * [Monthly analysis](#monthly-analysis)
+  {% endif %}
+  {% if 'day' in figures.keys() %}
+  * [Day of month analysis](#daily-analysis)
+  {% endif %}
+  {% if 'hour' in figures.keys() %}
+  * [Hour of day analysis](#hourly-analysis)
+  {% endif %}
+  {% if 'date' in figures.keys() %}
+  * [Date analysis](#date-analysis)
+  {% endif %}
+  {% if 'year' in figures.keys() %}
+  * [Yearly analysis](#yearly-analysis)
+  {% endif %}
+  {% if 'weekday' in figures.keys() %}
+  * [Day of week analysis](#weekday-analysis)
+  {% endif %}
```

Comment:
can you replace this with a loop like you've done below? I guess you might need to define a mapping for the text somewhere, which could be a net-loss of readability/simplicity. skip this comment if you don't know of a good way to do it.

Reply:
Sure we could add a mapping in Python and pass it to the template rendering.  I guess it depends where we want more of the customization to come from, the Python or jinja/html portions.  I guess my preference is to have more complexity on the Python side so I'm inclined to agree with you to move the mapping to Python, but maybe @alorenzo175 has thoughts?

Reply:
I agree that more complexity on the Python side is preferable.